src/App.vue

```diff
@@ -176,7 +176,7 @@ export default class App extends Mixins(
   @Getter isAuthenticated!: boolean
   @Getter isRoleStaff!: boolean
   @Getter isMobile!: boolean
-  @Getter isNewBusiness!: boolean
+  // @Getter isNewBusiness!: boolean
```

Comment:
This getter is already imported via a mixin so declaring it here displays an error in VS Code. However, I like to keep the declaration here so that it's explicit that this code file uses this getter.